pkg/operator/unsupportedcertrotation/config.go

```diff
@@ -0,0 +1,36 @@
+package unsupportedcertrotation
+
+import (
+	"time"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+)
+
+// GetCertRotationBase is helper method to read the unsupported cert rotation configuration
```

Comment:
```
// GetCertRotationScale  The normal scale is based on a day.  The value returned by this function is used to scale rotation durations instead of a day, so you can set it shorter.

```